mpsl/CHANGELOG.rst

```diff
@@ -19,6 +19,7 @@ Changes
 * Changed :c:func:`mpsl_fem_tx_power_split` function so the :c:member:`mpsl_tx_power_split_t.radio_tx_power` field contains a value supported by the RADIO peripheral.
   Previously the value needed to be adjusted before applying to the RADIO peripheral. (KRKNWK-14323).
 * Changed :c:struct:`mpsl_fem_gpiote_pin_config_t` and :c:struct:`mpsl_fem_gpio_pin_config_t` to require GPIO port address, port number and relative pin number instead of the absolute pin number (KRKNWK-11891).
+* Added production support for the 1-wire coexistence interface on the nRF52 Series (DRGN-16439).
```

Comment:
```suggestion
* Added production support for the one-wire coexistence interface on the Nordic nRF52 Series. (DRGN-16439)
```
If you could kindly fix the period placement in lines 18-21, I would be grateful. The parentheses with the JIRA issue should go after the period and not be closed with a period. Thanks.

Reply:
Fixed. I have updated the PR :slightly_smiling_face: